pkg/services/ngalert/models/receivers_test.go

```diff
@@ -320,7 +331,7 @@ func TestReceiver_Fingerprint(t *testing.T) {
 	completelyDifferentReceiver.Integrations[0].Config = IntegrationConfig{Type: completelyDifferentReceiver.Integrations[0].Config.Type} // Remove all fields except Type.
 
 	t.Run("stable across code changes", func(t *testing.T) {
-		expectedFingerprint := "ae141b582965f4f5" // If this is a valid fingerprint generation change, update the expected value.
```

Comment:
this fingerprint changed because we do not serialize all fields of settings anymore, only those that are unexpected (see `writeSettings` anonymous func)